pkg/apis/security/v1alpha1/types.go

```diff
@@ -154,12 +154,15 @@ type NetworkPolicyPort struct {
 	// If not specified, this field defaults to TCP.
 	// +optional
 	Protocol *v1.Protocol `json:"protocol"`
```

Comment:
why wasn't this changed to `omitempty` as well?

Reply:
Changed. Thx for pointing out.